mat/cholesky.go

```diff
@@ -294,53 +294,74 @@ func (c *Cholesky) RawU() Triangular {
 }
 
 // UTo extracts the n×n upper triangular matrix U from a Cholesky
-// decomposition into dst and returns the result. If dst is nil a new
-// TriDense is allocated.
-//  A = Uᵀ * U.
-func (c *Cholesky) UTo(dst *TriDense) *TriDense {
+// decomposition into dst and returns the result. If dst is empty
+// it is resized to be a upper-triangular n×n matrix. If dst is
```

Comment:
"When dst is non-empty, UTo panics if dst is not n×n or not Upper."

Reply:
Done.